tests/Dfc.CourseDirectory.WebV2.Tests/FeatureTests/Providers/EditProviderTypeTests.cs

```diff
@@ -77,14 +80,21 @@ public async Task Get_ValidRequestNoneProviderType_RendersExpectedOutput()
         [Theory]
         [InlineData(ProviderType.Apprenticeships, new[] { "apprenticeships" })]
         [InlineData(ProviderType.FE, new[] { "fe" })]
+        [InlineData(ProviderType.TLevels, new[] { "tLevels" })]
         [InlineData(ProviderType.Apprenticeships | ProviderType.FE, new[] { "fe", "apprenticeships" })]
+        [InlineData(ProviderType.Apprenticeships | ProviderType.TLevels, new[] { "apprenticeships", "tLevels" })]
+        [InlineData(ProviderType.FE | ProviderType.TLevels, new[] { "fe", "tLevels" })]
+        [InlineData(ProviderType.Apprenticeships | ProviderType.FE | ProviderType.TLevels, new[] { "fe", "apprenticeships", "tLevels" })]
+
         public async Task Get_ValidRequest_RendersExpectedOutput(
             ProviderType providerType,
             IEnumerable<string> expectedCheckedTestIds)
         {
             // Arrange
             var providerId = await TestData.CreateProvider(providerType: providerType);
 
+            var tLevelDefinitionIds = await Task.WhenAll(Enumerable.Range(0, 3).Select(_ => TestData.CreateTLevelDefinition()));
```

Comment:
I think `frameworkCode` & `progType` pairs are meant to be unique. If that's the case I think we should reflect that in the test data here.

Reply:
I'm not sure we know enough about the data at this stage to bother with this - we can look at this again once we're actually using that particular data.